### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by radiovisual**
I read the guidelines for contributing, but it seems overkill to open an issue in order to report a few typos in the Readme file, so I took the liberty of fixing the typos without opening an issue first. I hope that is alright!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,9 +79,9 @@ Building on Windows with Visual Studio requires version 2013 or higher.
 
 **Typical workflow:**
 
-0. Open an image and get an image object.
-0. Manipulate it.
-0. Save to disk / Send image buffer over network / etc.
+1. Open an image and get an image object.
```
Comment (EyalAr) [inline README.md]: I used 0's on purpose. Markdown transforms a list of zeros to a numbered list automatically.

File: README.md [hunk 2]
```diff
@@ -152,7 +152,7 @@ lwip.open('image.jpg', function(err, image){
 - GIF (no animations)
 
 Other formats may also be supported in the future, but are probably less urgent.
-Check the issues to see [which formats are planned to be supported](https://github.com/EyalAr/lwip/issues?labels=format+request&page=1&state=open).
+Check the issues to see [which formats may be supported in the future](https://github.com/EyalAr/lwip/issues?labels=format+request&page=1&state=open).
```
Comment (EyalAr) [inline README.md]: This phrasing implies that a certain format listed in the issues may never be supported. But if it's in the issues list, I do intend to support it at one point.
I prefer the old phrasing.

**Comment by EyalAr** [comment]
Thanks. Please see my comments.

**Comment by radiovisual** [comment]
Ok, no problem. I have updated this PR with a new commit to reflect your suggestions.

**Pull request closed without merging**